### DISCUSSION TITLE
HA Devices in Tuya

### DISCUSSION THREAD

**Discussion by dgoodwin430** [make-all/tuya-local#1647] [General]
Is it possible to add home assistant devices to tuya?

**Comment by make-all** [comment]
Off topic for this repository

**Reply by dgoodwin430** [reply]
Sorry, I didn’t think this was off topic. Maybe I wasn’t clear on my question. I am wanting to have home assistant devices show up in the Tuya/Smart Lift app. Basically so there can be 2-way communication.
